include/alpaka/acc/AccGenericSycl.hpp

```diff
@@ -133,7 +137,11 @@ namespace alpaka::trait
         static auto getAccDevProps(typename DevType<TAcc<TDim, TIdx>>::type const& dev) -> AccDevProps<TDim, TIdx>
         {
             auto const device = dev.getNativeHandle().first;
-            auto max_threads_dim = device.template get_info<sycl::info::device::max_work_item_sizes>();
+            auto const max_threads_dim
+                = device.template get_info<sycl::info::device::max_work_item_sizes<TDim::value>>();
```

Comment:
There is a breaking change in SYCL 2020 here. Maybe we should require the newer spec, or add a workaround for older SYCL versions.

Reply:
As discussed in the meeting, we will follow SYCL 2020.